react/components/Charts/commonProps.ts

```diff
@@ -0,0 +1,16 @@
+const commonDefaultProps = {
```

Comment:
Why it's not on the `constants` file?

Reply:
Because this props are common for many graphs, not only for line charts, that contains the `constants` file.